app/cdash/tests/ctest/CMakeLists.txt

```diff
@@ -153,7 +153,7 @@ add_php_test(timeline)
 add_php_test(nobackup)
 
 if (NOT CDASH_DB_TYPE STREQUAL "pgsql")
-  add_php_test(parallelsubmissions)
+  #add_php_test(parallelsubmissions)
 endif()
```

Comment:
Ok, sounds good.

Yes, I was the one who originally commented it out.  I've actually never been able to run that test locally.

I have that test commented out in the patch I have locally to get everything working with my ARM Mac, but I assumed it was user error on my part since it passes when running in the CI on master currently.